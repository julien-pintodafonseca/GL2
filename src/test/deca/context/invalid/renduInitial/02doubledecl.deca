{
	int x ;
	int y ;
	int z ;
	int y ;
	z=1 ;
}