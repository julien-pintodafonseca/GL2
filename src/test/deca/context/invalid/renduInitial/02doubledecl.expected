02doubledecl.deca:5:
